ApiClient/Tests/ApiClientTests.swift

```diff
@@ -121,4 +121,17 @@ class ApiClientTests: XCTestCase {
         waitForExpectations(timeout: 10.0) { (_) -> Void in }
     }
     
+    // Read API Key from API.plist (not included in source control, please create your own using the API Key we provided you with.)
```

Comment:
Why don't you just add the demo key to the test API.plist? That's what the demo key is for.